src/components/TimeseriesPreview/stories/full.md

```diff
@@ -0,0 +1,110 @@
+# Timeseries Preview
+
+<!-- STORY -->
+
+#### Description:
+
+Component displays data related to provided `timeseriesId` with latest datapoint available. Latest datapoint is refreshing constantly with interval provided via `updateInterval` prop.
+
+Component has embedded logic for fetching timeseries data and latest datapoint available for it. To make it work properly, component
+has to be wrapped into `ClientSDKProvider` component with defined client SDK instance. Fetching logic
+also can be redefined via `retrieveTimeseries` and `retrieveLatestDatapoint` props to retrieve timeseries data and latest datapoint respectively.
+
+#### Usage:
+
+```typescript jsx
+import 'antd/dist/antd.css';
+
+import React from 'react';
+import { TimeseriesPreview } from '@cognite/gearbox';
+
+function ExampleComponent(props) {
+  return <TimeseriesPreview timeseriesId={41852231325889} />;
+
+}
+```
+
+#### Available props:
+
+##### Required:
+
+| Property       | Description   | Type     | Default |
+| -------------- | ------------- | -------- | ------- |
+| `timeseriesId` | Timeseries id | `number` |         |
+
+##### Optionals:
+
+| Property                  | Description                                                                                                | Type                                                         | Default                  |
+| ------------------------- | ---------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------ | ------------------------ |
+| `color`                   | Rendered as a background color for the left side of the component                                          | `string`                                                     | `#6c65ee`                |
+| `dateFormat`              | Defines [date format](https://momentjs.com/docs/#/displaying/format/) to be applied on datapoint timestamp | `string`                                                     | `DD MMM YYYY - HH:mm:ss` |
+| `updateInterval`          | Refresh latest datapoint interval in ms                                                                    | `number`                                                     | `5000`                   |
+| `valueToDisplay`          | Datapoint that renders instead of latest datapoint. Pause fetching latest datapoint if provided            | `GetDoubleDatapoint \| GetStringDatapoint`                   |                          |
```

Comment:
```suggestion
| `valueToDisplay`          | Datapoint to be rendered instead of latest datapoint. Pause fetching latest datapoint if provided            | `GetDoubleDatapoint \| GetStringDatapoint`                   |                          |
```